openmc/mgxs/groups.py

```diff
@@ -77,6 +77,69 @@ def group_edges(self, edges):
         cv.check_greater_than('number of group edges', len(edges), 1)
         self._group_edges = np.array(edges)
 
+    @staticmethod
+    def get_energy_groups(name):
```

Comment:
Is there a more instructive name for this? Perhaps `get_common_energy_groups`?

Reply:
Yes, I like that name better as well.